odm2api/ODM2/services/readService.py

```diff
@@ -544,12 +544,30 @@ def getPeople(self, ids=None, firstname=None, lastname=None):
 
     def getAffiliations(self, ids=None, personfirst=None, personlast=None, orgcode=None):
         """
-        getAffiliations(self, ids=None, personfirst=None, personlast=None, orgcode=None)
-        * Pass nothing - returns a list of all Affiliation objects
-        * Pass a list of AffiliationID - returns a single Affiliation object
-        * Pass a First Name - returns a single Affiliation object
-        * Pass a Last Name - returns a single Affiliation object
-        * Pass an OrganizationCode - returns a Affiliation object
+        Retrieve a list of ODM2 Affiliation objects.
```

Comment:
Based on the behavior described in the old docstring, I'd change this to: "Retrieve an Affiliation object or a list of such objects." But please confirm whether a request that returns one object, returns a list with one element or the object proper. Adjust the above string as appropriate.

Reply:
The previous docstring seems to be incorrect, the function always returns a list...

Reply:
Ok. Glad you checked it!

I would just remove "ODM2" from the string. It's obvious that odm2api query methods all return ODM2 objects!